index.js

```diff
@@ -311,18 +311,57 @@ function cms(loopbackApplication, options) {
     });
   });
 
+  if (config.public.isUnsafeUpsert) {
```

Comment:
Does this need to be on public? Does it work as config.isUnsafeUpsert?

And can we rename to config.enableUnsafeUpsert?